
const {ValidationError} = require('./functions/common/Error');

// try {
//   throw new ValidationError('This is error...');
// } catch(e) {
//   console.log(e.name);
//   console.log(e.message);
//   console.log(e.code);
// }

// try {
//   throw new Error('This is error');
// } catch(e) {
//   console.log(e.name);
//   console.log(e.message);
//   console.log(e.code);
// }